src/Routes/RoutesCommon.tsx

```diff
@@ -13,7 +13,7 @@ import { SwapIcon } from 'components/Icons/SwapIcon'
 import { TxHistoryIcon } from 'components/Icons/TxHistory'
 import { WalletIcon } from 'components/Icons/WalletIcon'
 import { assetIdPaths } from 'hooks/useRouteAssetId/useRouteAssetId'
-import { RFOX } from 'pages/RFOX/RFOX'
+import { RfoxDashboard } from 'pages/RFOX/RfoxDashboard'
```

Comment:
Can we keep this as RFOX for consistency? Dashboard is implied here, see e.g lending or THORChain LP for naming